app/models/developer_key.rb

```diff
@@ -101,6 +101,14 @@ def account_name
     account.try(:name)
   end
 
+  def access_token_count
+    self.access_tokens.count
```

Comment:
@tpickett66 Actually, not quite. When checking the test I wrote for `access_token_count`, I found that when a new token is created, the `developer_keys.access_token_account` field is `NULL`, not `0`. 

I could fix this in a couple of ways:
- setting the default value for `developer_keys.access_token_count` to `0` and making it `NOT NULL`; or
- by calling `DeveloperKey.reset_countners(self.id, :access_tokens)` (probably in an `after_create` method).

I'm leaning towards the default value unless you've got a different suggestion.